FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+                }
+                else
+                {
+                    // Handle errors or unsuccessful find.
+                }
+                return S_OK;
+            }).Get()));
+
+    // Note: In this example, navigation through find results (FindNext/FindPrevious)
+    // and stopping the find operation (StopFind) are assumed to be handled by
+    // custom UI elements and user interaction, not directly in code here.
+    // User could then connect functions such as FindNext, FindPrevious, and StopFind
+    // to corresponding custom UI elements.
+
+    return true;
+}
+//! [ExecuteFindWithCustomUI]
+```
+#### .NET C#
+```csharp
+//! [ConfigureAndExecuteFindWithDefaultUI]
+async Task ConfigureAndExecuteFindWithDefaultUIAsync(string findTerm)
+{
+    try
+    {
+        // Check if the webView is already initialized and is an instance of CoreWebView2.
+        if (webView.CoreWebView2 == null)
+        {
+            throw new InvalidOperationException("WebView2 is not initialized.");
+        }
+
+        // Initialize the find configuration with specified settings.
+        var findConfiguration = new CoreWebView2FindConfiguration
+        {
+            FindTerm = findTerm,
+            IsCaseSensitive = false,
+            ShouldMatchWord = false,
+            FindDirection = CoreWebView2FindDirection.Forward
+        };
+
+        // By default Find will use the default UI and highlight all matches. If you want different behavior
+        // you can change the SuppressDefaultDialog and ShouldHighlightAllMatches properties here.
+
+        // Start the find operation with the specified configuration.
+        await webView.CoreWebView2.Find.StartFindAsync(findConfiguration);
+
+        // End user interaction is handled via UI.
+    }
+    catch (Exception ex)
+    {
+        // Handle any errors that may occur during the find operation.
+        Console.WriteLine($"An error occurred: {ex.Message}");
+    }
+}
+//! [ConfigureAndExecuteFindWithDefaultUI]
+```
+
+```csharp
+//! [ConfigureAndExecuteFindWithCustomUI]
+async Task ConfigureAndExecuteFindWithCustomUIAsync(string findTerm)
+{
+    try
+    {
+        // Check if the webView is already initialized and is an instance of CoreWebView2.
+        if (webView.CoreWebView2 == null)
+        {
+            throw new InvalidOperationException("WebView2 is not initialized.");
+        }
+
+        // Initialize the find configuration with specified settings.
+        var findConfiguration = new CoreWebView2FindConfiguration
+        {
+            FindTerm = findTerm,
+            IsCaseSensitive = false,
+            ShouldMatchWord = false,
+            FindDirection = CoreWebView2FindDirection.Forward
+        };
+
+        // Specify that a custom UI will be used for the find operation.
+        webView.CoreWebView2.Find.SuppressDefaultDialog = true;
+        webView.CoreWebView2.Find.ShouldHighlightAllMatches = true;
```

Comment:
It's not obvious at first glance which things are "find configuration" things and which are "find things", particularly since "ShouldHighlightAllMatches" is **configuring** the default Find UI!

Maybe call one of them FindRequest and the other FindUI?

Reply:
Please fix sample code. These properties are now on the Configuration / Options object.